cylc/flow/config.py

```diff
@@ -1561,7 +1574,7 @@ def configure_workflow_state_polling_tasks(self):
                     ('alt-cylc-run-dir', ' --%s=%s')]:
                 if rtc['workflow state polling'][key]:
                     comstr += fmt % (key, rtc['workflow state polling'][key])
-            script = "echo " + comstr + "\n" + comstr
+            script = "set -x; " + comstr + "\n" + comstr
```

Comment:
Sorry? I was happy with keeping the echo in the end; and if you are going to do this, you only want 1 copy of `comstr`.

Reply:
No I wasn't going to do that. I think I was experimenting with your suggesting and must have committed it 🤦‍♂️

Reply:
(Fixed)